### ISSUE TITLE
[a11y]: text area - readOnly mode

### ISSUE THREAD

**Issue by LyndseyBu** [carbon-design-system/carbon#19481]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.81.0

### React version

20

### Automated testing tool and ruleset

Colour contrast analyser

### Description

Our AVT test team have identified an issue with the text area border when in readOnly mode.
The bottom border which is provided to differentiate the form field has contrast ratio less than 3:1 . Current ratio 1.3:1
![Image](https://github.com/user-attachments/assets/df351f84-8bd7-4e32-b259-1d1ad31fbf88)

### WCAG 2.1 Violation

1.4.11 Non-Text Contrast Level of Compliance: AA

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-textarea--default&args=readOnly:!true

### Steps to reproduce

using storybook, select the default TextArea and use the controls to set readOnly=true
check the colour contrast between the border on the bottom of the text are and the background

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by annawen1** [comment]
Hi @LyndseyBu because the `TextArea` component on readOnly mode is inactive, I believe it doesn't violate 1.4.11 Non-Text Contrast. In the WCAG 2.2 docs, it states under 'User Interface Components' section: 

"Unless the control is **inactive**, any visual information provided that is necessary for a user to identify that a control is present and how to operate it must have a minimum 3:1 contrast ratio with the adjacent colors."

https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

**Comment by LyndseyBu** [comment]
Closing as the AVT team are now in agreement that as it is read only mode it does not violate 1.4.11 Non-Text Contrast
